Lib/test/ann_module5.py

```diff
@@ -0,0 +1,8 @@
+from __future__ import annotations
```

Comment:
Please add a comment or docstring at the top describing what this is for. Something similar to your docstring in ``ann_module6`` is good.